istio/src/main/java/com/alibaba/nacos/istio/util/IstioCrdUtil.java

```diff
@@ -40,7 +40,7 @@
 
     public static final String VALID_DEFAULT_GROUP_NAME = "DEFAULT-GROUP";
 
-    private static final String SUFFIX_DOMAIN = "nacos";
+    private static final String ISTIO_HOSTNAME = "istio.hostname";
```

Comment:
I prefer to define a configuration var in IstioConfig class to controll domain suffix for all nacos services.

Reply:
I agree. At the time I was also thinking about whether it would be better to have a unified suffix. I changed it.

Reply:
I didn't find the configuration about the domain suffix. Do I missing something?
May be like this.
```
@Value("${nacos.istio.domain.suffix:nacos}")
private String domainSuffix;
```